python/semantic_kernel/core_skills/text_memory_skill.py

```diff
@@ -19,9 +17,9 @@ class TextMemorySkill:
     DEFAULT_RELEVANCE = 0.75
 
     # @staticmethod
-    @sk_function("Recall a fact from the long term memory")
-    @sk_function_name("recall")
-    @sk_function_input(description="The information to retrieve")
+    @sk_function("Recall a fact from the long term memory", 
```

Comment:
Thank for looking this! I will fix the issue.

Reply:
@jjhenkel  Added the missing `self`,
on my side the memory seems to work now:
```
[...]
Setting up a chat (with memory!)
Begin chatting (type 'exit' to exit):

User:> where is my family located
ChatBot:>  Your family is from New York.
User:> what are the countries that I traveled to five times
ChatBot:>  You visited France and Italy five times since 2015.
User:> what are the countries that I traveled  to in 1993
ChatBot:>  I don't know.```